README.md

```diff
@@ -59,6 +60,27 @@ cudnn.verbose = true -- this prints out some more verbose information useful for
 ```
 by default, `cudnn.verbose` is set to `false`.
 
+### Conversion between `cudnn` and `nn`
+
+Conversion is done by `cudnn.convert` function which takes a network and backend arguments and goes over
```

Comment:
Could you also mention (or add an example) that you can convert to both `nn` and `cudnn`  backends ? It's not clear from the README that it's possible to go from `cudnn` to `nn`.